oak/client/authorization_bearer_token_metadata.h

```diff
@@ -14,34 +14,30 @@
  * limitations under the License.
  */
 
-#ifndef OAK_CLIENT_PER_CALL_POLICY_H_
-#define OAK_CLIENT_PER_CALL_POLICY_H_
+#ifndef OAK_CLIENT_AUTHORIZATION_BEARER_TOKEN_METADATA_H_
+#define OAK_CLIENT_AUTHORIZATION_BEARER_TOKEN_METADATA_H_
 
 #include "include/grpcpp/grpcpp.h"
-#include "oak/common/nonce_generator.h"
 
 namespace oak {
 
-namespace {
-constexpr size_t kPerCallNonceSizeBytes = 32;
-}  // namespace
-
-// This class generates a fresh per-call nonce and injects it to the gRPC metadata of each call as
-// an authorization bearer token.
+// This class injects the provided authorization bearer token to the gRPC metadata of each call.
+//
+// This is used to fulfil policies that refer to such tokens.
```

Comment:
nit: this comment doesn't really belong here &ndash; it describes how some other code elsewhere might use this class, rather than what the class does.